src/TLuaInterpreter.cpp

```diff
@@ -418,62 +418,80 @@ int TLuaInterpreter::resetProfile( lua_State * L )
 
 
 // cursorPositionInLine = select( text ) if not found -1
-int TLuaInterpreter::select( lua_State * L )
+// Was called select but that may clash with the Lua built-in command with the
```

Comment:
The fact that it was this:

```
-    lua_register( pGlobalLua, "selectString", TLuaInterpreter::select );
+    lua_register( pGlobalLua, "selectString", TLuaInterpreter::selectString );
```

Means it did not clash, as "selectString" is what was registered with Lua. So the comment is unnecessary, but I welcome the change for consistency anyway.